manifest/provider/waiter.go

```diff
@@ -174,6 +181,8 @@ func (w *FieldWaiter) Wait(ctx context.Context) error {
 		if done {
 			return err
 		}
+
+		time.Sleep(1 * time.Second) // lintignore:R018
```

Comment:
We were hammering the API here so I added this sleep.

Reply:
Nice! This should work great!

At some point we could make it extra spiffy by leveraging some of the exponential back-off helpers in API machinery and client-go:
https://pkg.go.dev/k8s.io/apimachinery/pkg/util/wait
https://pkg.go.dev/k8s.io/client-go/util/retry

Reply:
Let's just add a TODO so that this doesn't get forgotten (I will for sure).
```suggestion
              // TODO: implement with exponential back-off.
		time.Sleep(1 * time.Second) // lintignore:R018
```